### PR TITLE
Async redux

### PR DESCRIPTION
**Pull request by vadimshut**
1. Task: https://d14aa3aqnm30ti.cloudfront.net/docs/module-4/home-task/task
2. Screenshot:
  
image
image

3. Deploy:-
4. Done 20.07.2022 / deadline 18.07.2022
5. Score: 30 / 30

### CODE REVIEW

File: src/store/dataFromStore.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+const getRegistrationData = (state) => ({
+	registerSuccess: state.userReducer.registerSuccess,
+	registerError: state.userReducer.registerError,
+});
+
+const getAuthors = (state) => state.authorsReducer.authors;
+const getCourses = (state) => state.coursesReducer.courses;
+const getCourseForUpdate = (state) => state.coursesReducer.courseForUpdate;
+
+export { getAuthData, getRegistrationData, getAuthors, getRole, getCourses, getCourseForUpdate };
```
Comment (ArtemOsadchuck) [inline src/store/dataFromStore.js]: please rename this file to "selectors"

File: src/store/authors/authors.js [hunk 2]
```diff
@@ -9,28 +9,26 @@ const authorsSlice = createSlice({
...
-		},
-		deleteAuthors(state, action) {
-			state.authors.push(action.payload);
-		},
-		updateAuthors(state, action) {
-			state.authors.push(action.payload);
-		},
+		// addAuthor(state, action) {
+		// 	state.authors.push(action.payload);
+		// },
```
Comment (ArtemOsadchuck) [inline src/store/authors/authors.js]: redundant comment

Comment (vadimshut) [reply src/store/authors/authors.js]: delete commented code

File: src/store/dataFromStore.js (hunk 1 above)
Comment (vadimshut) [reply src/store/dataFromStore.js]: rename file

**Pull request merged**